packages/next/src/lib/helpers/get-reserved-port.ts

```diff
@@ -0,0 +1,94 @@
+export const KNOWN_RESERVED_PORTS = {
```

Comment:
```suggestion
// https://fetch.spec.whatwg.org/#port-blocking
export const KNOWN_RESERVED_PORTS = {
```

Let's reference the spec.